### ISSUE TITLE
ParticlePlot Documentation link broken

### ISSUE THREAD

**Issue by npolys** [jamesleesaunders/d3-x3d#195]
on the githhub landing page

https://github.com/jamesleesaunders/d3-x3d

:-]

also the Pollen.data set can be attributed as :
"The Bell Labs Pollen data set: In the 1980s, the ASA (American Statistical Association) held a data competition challenged contestants to find as many features as possible in a 5 dimensional data set of 4000 cases. Among other important (known) features in the data set, in a scatterplot of three of the dimensions, the word ’EUREKA’ was clearly apparent. Participants had a deadline, but were not timed; the number of insights or features identified by the contestants was used to judge the winner(s). (Handbook of Data Vis pg179)"
- Dahshan & Polys 2021:

Available at 
https://www.openml.org/d/529

**Comment by npolys** [comment]
specifically interested in
if the ParticlePlot supports data mapping to the scale (of each point) 
and to the transparency field of the (RGB [A] ) Color ? :-]

**Comment by jamesleesaunders** [comment]
Hi Nicholas, 
Thanks for highlighting the broken particle plot link. I have fixed this.

I have also added credit as suggested for the polled data set beneath each example:
https://raw.githack.com/jamesleesaunders/d3-x3d/master/examples/X3DOM/chart/ParticlePlot.html
As well as note at the bottom of the README credits. Thank you!

I think it should support RGBA (transparency) out of the box by passing a RGBA in to the color attribute. I will have a play and see how this behaves.
